src/iOS/Services/iOSPushNotificationHandler.cs

```diff
@@ -83,8 +83,6 @@ private static string DictionaryToJson(NSDictionary dictionary)
         public void WillPresentNotification(UNUserNotificationCenter center, UNNotification notification, Action<UNNotificationPresentationOptions> completionHandler)
         {
             Debug.WriteLine($"{TAG} WillPresentNotification {notification?.Request?.Content?.UserInfo}");
-
-            OnMessageReceived(notification?.Request?.Content?.UserInfo);
```

Comment:
oh shoot! Good catch this was by mistake! i actually wanted to delete on the response method below, to avoid infinite looping, but since i decided not to inject any info in the local notification, it won't trigger the `OnMessageReceived` anyway.